### PR TITLE
fix(plugin-meetings): framerate multiplied by 100

### PR DESCRIPTION
**Pull request by SomeBody16**
# COMPLETES [WEBEX-396717](https://jira-eng-gpk2.cisco.com/jira/browse/WEBEX-396717)

## This pull request addresses

The frame rate of metrics that is from CLP automateddiagnostics for WEB APP is not unified with Webex APP. The frame rate of the Webex APP is multiple 100, but the WEB APP is not. 

## by making the following changes

Multiply frame related stats by 100

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

- samples app correctly sends stats multiplied by 100
- unit tests

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js [hunk 1]
```diff
@@ -1925,7 +1925,7 @@ describe('plugin-meetings', () => {
                 rtpPackets: 0,
                 ssci: 0,
                 transmittedBitrate: 133.33333333333334,
-                transmittedFrameRate: 0,
+                transmittedFrameRate: 2,
```
Comment (brycetham) [inline packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js:1928]: Doesn't really matter since we rewrote this test for the StatsAnalyzer move to internal-media-core, but just wondering what's the purpose of this change?

**Review by brycetham — APPROVED** [review]
Looks good. Just had a small question.

**Review by k-wasniowski — APPROVED** [review]

File: packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js (hunk 1 above)
Comment (antsukanova) [reply packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js:1928]: @brycetham This change needed due to the total frames sent in a minute, which equals 1, being multiplied by 100, divided by 60, and rounded, resulting in a value of 2. Therefore, we need to update this in order to ensure that the tests work correctly for the SDK.

**Review by antsukanova — COMMENTED** [review]
LGTM

**Review by antsukanova — APPROVED** [review]

File: packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js (hunk 1 above)
Comment (brycetham) [reply packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js:1928]: Ah, that's right. So this is the equivalent of 0.02 frames per second, which seems to be correct.

**Review by mkesavan13 — APPROVED** [review]
